frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/storage-review.tsx

```diff
@@ -50,6 +50,13 @@ const StorageReviewFirehose: React.FC<StorageReviewFirehoseProps> = ({
   persistentVolumeClaims,
   storageClasses,
 }) => {
+  const headers = [
+    { title: 'Name', transforms: [cellWidth(10)] },
+    { title: 'Interface', transforms: [cellWidth(10)] },
+    { title: 'Size', transforms: [cellWidth(10)] },
+    { title: 'Storage Class', transforms: [cellWidth(10)] },
```

Comment:
we need VolumeMode and AccessModes here as well. 

I wonder if it would make sense to have the `Source` here too?

Reply:
I added the `VolumeMode` and `AccessModes` fields. I don't think it would hurt to add `Source`, but I forgot to add it in this round of changes. I'll add it in the next round.

Reply:
Source added. =)